commit 950f219ed287358df8c128f7e22989177a8de47c
Author: Arnout Engelen <arnout@bzzt.net>
Date:   Mon Aug 25 15:55:44 2025 +0200

    man: simplify vmautoactivation(7)
    
    Previously this was hard-coded to: "Autoactivation commands use a number
    of temp files in /run/lvm (with the expectation that /run is cleared
    between boots.)"
    
    Since c1bfc8737f08bf7558b2d788e9756f895cd9eaaa it was made more generic,
    but on some systems this logic leads to "Autoactivation commands use a
    number of temp files in /run/lvm (with the expectation that /var/run
    is cleared between boots)." which I'd say adds more confusion than it
    solves.

diff --git a/man/lvmautoactivation.7_main b/man/lvmautoactivation.7_main
index e55943b29..9d429055c 100644
--- a/man/lvmautoactivation.7_main
+++ b/man/lvmautoactivation.7_main
@@ -175,9 +175,7 @@ is reserved for udev output.)
 .
 Autoactivation commands use a number of temp files in
 .I #DEFAULT_RUN_DIR#
-(with the expectation that
-.I #DEFAULT_PID_DIR#
-is cleared between boots).
+(with the expectation that it is cleared between boots).
 .
 .TP
 .B pvs_online
